meteor/lib/api/rundownLayouts.ts

```diff
@@ -163,6 +192,41 @@ export namespace RundownLayoutsAPI {
 	registry.registerRundownHeaderLayouts(RundownLayoutType.RUNDOWN_HEADER_LAYOUT, {
 		supportedFilters: [],
 	})
+	registry.registerRundownHeaderLayouts(RundownLayoutType.DASHBOARD_LAYOUT, {
+		filtersTitle: 'Layout Elements',
```

Comment:
Because the registry lives in the lib shared between client and server I can't access a TFunction directly, instead I've wrapped the `filtersTitle` property in a TFunction that is passed when the client requests the manifest of registered layouts https://github.com/tv2/tv-automation-server-core/commit/018fd9a728cad8843c13efe531a66b857c4bfb27